app/email_batching_test.go

```diff
@@ -281,15 +252,14 @@ func TestCheckPendingNotificationsCantParseInterval(t *testing.T) {
 
 	// notifications should not be sent 1s after post was created, because default batch interval is 15mins
 	job.checkPendingNotifications(time.Unix(10001, 0), func(string, []*batchedNotification) {})
-	if job.pendingNotifications[th.BasicUser.Id] == nil || len(job.pendingNotifications[th.BasicUser.Id]) != 1 {
-		t.Fatal("shouldn't have sent queued post")
-	}
+	require.NotNil(t, job.pendingNotifications[th.BasicUser.Id])
+	require.Len(t, job.pendingNotifications[th.BasicUser.Id], 1, "shouldn't have sent queued post")
 
 	// notifications should be sent 901s after post was created, because default batch interval is 15mins
 	job.checkPendingNotifications(time.Unix(10901, 0), func(string, []*batchedNotification) {})
-	if job.pendingNotifications[th.BasicUser.Id] != nil || len(job.pendingNotifications[th.BasicUser.Id]) != 0 {
-		t.Fatal("should have sent queued post")
-	}
+
+	require.Nil(t, job.pendingNotifications[th.BasicUser.Id], "should have sent queued post")
+	require.Len(t, job.pendingNotifications[th.BasicUser.Id], 0, "should have sent queued post")
```

Comment:
Here I think is just enough the `require.Nil` because if that is nil the `Len` is going to be always  `0`.